src/core_plugins/kibana/server/routes/api/scripts/register_languages.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerLanguages(server) {
+  server.route({
+    path: '/api/kibana/scripts/languages',
+    method: 'GET',
+    handler: function (request, reply) {
+      const boundCallWithRequest = _.partial(server.plugins.elasticsearch.callWithRequest, request);
+
+      return boundCallWithRequest('cluster.getSettings', {
```

Comment:
I think I did it here purely out of habit, I'll switch it to using callWithRequest directly to make it less confusing.

Usually it's for improving testability of libraries that need to make calls to ES. Instead of having the lib require a hapi request object, and then having to mock out that request object, you just have to worry about mocking this one function that gets passed in. In this case we're not using any external libs though, so it doesn't make much sense.